### PR TITLE
Add helm dep update as well as clean up the Helm doc

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

File: install/kubernetes/helm/istio/README.md [hunk 1]
```diff
@@ -38,31 +38,31 @@ The chart deploys pods that consume minimum resources as specified in the resour
...
+   $ helm init --service-account tiller
+   ```
+
+1. Set and create the namespace where Istio was installed:
+   ```
+   $ NAMESPACE=istio-system
+   $ kubectl create ns $NAMESPACE
+   ```
+
+1. Install Istio’s [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: @sdake do you mind update here a bit by highlighting this is only required by the helm which < 2.10 , we have updated this in istio.io but not here yet https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#installation-steps

**Comment by sdake** [comment]
rebased to fix circleci.

**Pull request merged**